### PR TITLE
Issue Template Review

### LINKED ISSUE #32: Need for a Bit Project-wide Issue Template
### Issue Name: 

Bit Project Issue Template

#### Context: 

Previously, there was no issue template for members to use whenever they were creating an issue that needed to be addressed. To organize and better keep our Github repository clean, we need to create a standard form for users to fill out.

#### Type: 

Issue Documentation Addition

#### Actual Issue: 

Need for Issue Template

### Desired Outcome: 

The issue template should address the problem/issue, give any context or information that would help a user/open-source member contribute and fix it, and also clearly label any examples/hints that would help fix the issue.

### (Optional) Hint/Example:

None.

### Possible Solutions: 

Some other users have created some pretty advanced issue templates (see https://github.com/stevemao/github-issue-templates), we could use this for inspiration or as an example.

### Other Details:

None.

### PR DESCRIPTION
**Pull request by beaubhp**
@emsesc I believe we went over the Issue Template already but might as well confirm it with @lazyplatypus. Also, thoughts about having one or two templates (as I saw with the issue template article they had one for bugs and one for additional features).

### CODE REVIEW

**Review by MinyMeep1 — COMMENTED** [review]
I don't see any issues with this exact template but maybe in the future, it could be edited depending on the comments we get from fellow contributors.

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+The following is an template for members to use whenever creating an issue for members or the open-source community. Each section should be filled out unless stated as optional. For problem type, state whether the issue is targetting an error/bug, documentation fix, need for additional information, etc. If you have any other questions about creating or contributing, please checkout the *"how to contribute"* Github issue [here](https://github.com/bitprj "here") (link coming soon).
+
+**Template:**
+
+---
+
+
+### Issue Name:
+
+### Problem:
```
Comment (lazyplatypus) [inline ISSUE_TEMPLATE.md:12]: I think under each "heading" we could add a line about what the naming convention/what should be under each one. For example, is a problem being "slack apps docs suck" probably isn't what we want.

**Review by lazyplatypus — CHANGES_REQUESTED** [review]
i think we still need to elaborate in the issue template

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (beaubhp) [reply ISSUE_TEMPLATE.md:12]: Good idea. I added additional "headings" under each segment @lazyplatypus.

**Comment by lazyplatypus** [comment]
This is ready to merge. however, i need you to move to file to .github/ISSUE_TEMPLATE/ and make this one of a few templates we will add. I think we should have different issue templates for each issue? What are your thoughts?

**Comment by beaubhp** [comment]
So are you suggesting one for features and one for bugs? Something like that? @lazyplatypus

**Comment by lazyplatypus** [comment]
Yes!

**Comment by lazyplatypus** [comment]
follow instructions here -> https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/configuring-issue-templates-for-your-repository

**Comment by beaubhp** [comment]
Okay, so what should I set this one up as? Bug? Feature? Custom? @lazyplatypus

**Comment by lazyplatypus** [comment]
a bug make a seperate feature one and create another PR after this

**Comment by lazyplatypus** [comment]
a bug

**Comment by lazyplatypus** [comment]
Duplicate of #39. Closing for now

**Review by emsesc — APPROVED** [review]
Everything looks great! I'll look over the other templates once they're committed. I would recommend adding a markdown file explaining when to use each template or clarifying this at the top of each template. @lazyplatypus thoughts?

**Pull request closed without merging**